tests/data_sources/satellite/test_satellite_data_source.py

```diff
@@ -71,6 +71,7 @@ def test_get_example(sat_data_source, x, y, left, right, top, bottom):  # noqa:
     assert np.isclose(top, sat_data.y_osgb.values[-1])
     assert len(sat_data.x_osgb) == pytest.IMAGE_SIZE_PIXELS
     assert len(sat_data.y_osgb) == pytest.IMAGE_SIZE_PIXELS
+    assert len(sat_data.x_osgb.shape) == 1
```

Comment:
if the dims are (example, x_osgb_index) then wouldn't `len(sat_data.x_osgb.shape) == 2` (not 1)?

Reply:
that particular test is for one example, so it should just be `1`, i.e 1D. Of course when its a batch of data it will have `length(shape) == 2`, i.e 2D.

Reply:
ah, got it, thanks!  sorry, I got myself confused!